lib/src/main/kotlin/com/squareup/cycler/Recycler.kt

```diff
@@ -576,7 +576,6 @@ class Recycler<I : Any> internal constructor(
       crossinline block: Config<I>.() -> Unit
     ): Recycler<I> {
       val view = RecyclerView(context)
-      view.id = R.id.recyclerView
```

Comment:
If you don't set the ID, don't use lose view persisence?

Reply:
Yes, the container should set the id themselves via `recycler.view.id = ...`.
But if I want to set it here it needs to be a resource (and it won't work by itself if there are two recyclers in the hierarchy).

Reply:
So then it seems like this PR should include a doc change explaining that. Should `create()` have an `id` argument? Maybe required, maybe it defaults to -1 (which means you skip the set)?

Reply:
Sounds good. I'll add the optional id argument. And check the docs, although I'm not sure there's mention of it.

Reply:
Maybe nullable instead of `-1` default?